src/consensus/pbft/libbyz/test/test_ledger_replay.cpp

```diff
@@ -376,7 +376,9 @@ TEST_CASE("Test Ledger Replay")
         REQUIRE(
           store->deserialise_views(corrupt_entry, false, nullptr, &tx) ==
           kv::DeserialiseSuccess::PASS_PRE_PREPARE);
-        pbft::GlobalState::get_replica().playback_pre_prepare(tx);
+        REQUIRE_THROWS_AS(
```

Comment:
Why is this required?

Reply:
Because we now throw if when we are playing back the merkle roots do not match

Reply:
In normal execution, when merkle roots do not match we expect to view change. When we are playing back we can't view change so the only solution is for the node to start again.